en/ui-system/components/editor/ui-transform.md

```diff
@@ -12,6 +14,29 @@ Click the __Add Component__ button at the bottom of the __Inspector__ panel and
 | AnchorPoint | The anchor position of UI rectangle.
 | Priority | The priority of UI nodes, sorted in the parent node. The order of the Canvas node is not affected by this property.
 
+### change the size and anchor point through script
```

Comment:
change the size and anchor point through script
-->
Change the size and anchor point in script